modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/RelationshipObjectFieldBusinessType.java

```diff
@@ -194,6 +196,8 @@ public Object getValue(
 				if (_log.isDebugEnabled()) {
 					_log.debug(exception);
 				}
+
+				objectEntryNotFound = exception;
```

Comment:
I think the exception may be any exception, not only the not found, so I think the name chosen is not correct